SNIP-25.md

```diff
@@ -0,0 +1,87 @@
+# SNIP-25 - Improved privacy for [SNIP-20](/SNIP-20.md) tokens
```

Comment:
This is not specific to SNIP-25, but should we add a definition of "SHOULD", "MAY", and "MUST" as they are used in this context?